models/class.metrilo-product-data.php

```diff
@@ -17,9 +17,9 @@ public function get_products($chunk_id)
         $products = [];
         $product_ids = $this->db_connection->get_col(
             "SELECT id FROM {$this->db_connection->posts}
-    WHERE post_type = 'product'
-    limit {$this->chunk_items}
-    offset {$chunk_id}"
+            WHERE post_type = 'product'
+            limit {$this->chunk_items}
+            offset {$chunk_id}"
```

Comment:
I have a question - `$this->db_connection->get_col` is this function SQL injection safe? I see that we build the query string from multiple variables.